core/drivers/pmd.h

```diff
@@ -127,7 +127,7 @@ class PMDPort final : public Port {
 
   LinkStatus GetLinkStatus() override;
 
-  int UpdateConf(const Conf &conf) override;
+  CommandResponse UpdateConf(const Conf &conf) override;
```

Comment:
Are we sure this change won't break any outside drivers?

Reply:
Yes :)